src/libs/OptionsListUtils.js

```diff
@@ -215,7 +215,7 @@ function createOption(personalDetailList, report, draftComments, {
         login: !hasMultipleParticipants ? personalDetail.login : null,
         reportID: report ? report.reportID : null,
         isUnread: report ? report.unreadActionCount > 0 : null,
-        hasDraftComment,
+        hasDraftComment: !!reportDraftComment && reportDraftComment.length > 0,
```

Comment:
you don't need the `!!` here right. You could just do,
```
reportDraftComment && reportDraftComment.length > 0
```

Reply:
This is not true and completely broke the app for me 😃 

![image](https://user-images.githubusercontent.com/12156624/124727289-42081580-df17-11eb-8d5c-964d30929362.png)

src/pages/home/sidebar/OptionRow.js
```jsx
                            {option.hasDraftComment && (
                                <View style={styles.ml2}>
                                    <Icon src={Pencil} height={16} width={16} />
                                </View>
                            )}
```

I have some empty `""` draft comment somewhere and it just crashes the app

The cast to boolean is necessary, though I would go with the explicit 

```jsx
Boolean(reportDraftComment) && reportDraftComment.length > 0
```

or

```jsx
Boolean(reportDraftComment && reportDraftComment.length > 0)
```

I don't know why I have an empty comment but it seems plausible and since `reportDraftComment` is a string it should be casted to prevent accidents like this